_code-samples/get-started/go/base/rpc/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+    "github.com/Peersyst/xrpl-go/xrpl/queries/utility"
+    "github.com/Peersyst/xrpl-go/xrpl/rpc"
+)
+
+func main() {
+    // Define the network client configuration
+    cfg, err := rpc.NewClientConfig("https://s.altnet.rippletest.net:51234/")
```

Comment:
Shouldn't we add the `WithFaucetProvider()` function here since we are connecting to Testnet?

```suggestion
    cfg, err := rpc.NewClientConfig(
        "https://s.altnet.rippletest.net:51234/",
        rpc.WithFaucetProvider(faucet.NewTestnetFaucetProvider()),
    )
```